pkg/kdump/config-client.js

```diff
@@ -47,7 +47,7 @@ export class ConfigFile {
 
     close() {
         if (this._fileHandle) {
-            this._fileHandle.remove();
+            this._fileHandle.close();
```

Comment:
Looking at the [docs](https://cockpit-project.org/guide/latest/cockpit-file) cockpit.file("foo").watch() has a `remove` function.

As for example in our test `pkg/base1/test-file.js`:

```
      const file = cockpit.file(dir + "/foobar");
      const watch = file.watch(changed);
      watch.remove();
```

So is our code above wrong here, or does cockpit.file().close() also remove the watchers?

Reply:
As in, should we assign `this._watchhandle = file.watch()` and `remove()` that?

Reply:
I thought it might have come from the `watch.remove()` but there's no history in git to prove it wrong/right.
I think `close()` has enough logic to take care of the watches: https://github.com/cockpit-project/cockpit/blob/main/pkg/lib/cockpit.js#L3800